bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -405,12 +409,27 @@ public int getChannelCount() {
   /**
    * <p>Getter for the field <code>instanceName</code>.</p>
    *
-   * @return a {@link com.google.cloud.bigtable.grpc.BigtableInstanceName} object.
+   * @return a {@link BigtableInstanceName} object.
    */
   public BigtableInstanceName getInstanceName() {
     return instanceName;
   }
 
+  /**
+   * <p>Getter for the field <code>instanceName</code>.</p>
+   *
+   * @return a {@link RequestContext} object.
+   */
+  public RequestContext getRequestContext() {
```

Comment:
This context can only be used for the data client. Maybe rename as such. Also, the docs are a bit off

Reply:
Also consider keeping the RequestContext in client implementation (maybe BigtableSession?) , since its not really a configureable option

Reply:
I moved `RequestContext` to `BigtableSession` and renamed.